include/gz/math/SphericalCoordinates.hh

```diff
@@ -73,6 +81,21 @@ namespace gz
       /// \param[in] _type SurfaceType specification.
       public: explicit SphericalCoordinates(const SurfaceType _type);
 
+      /// \brief Constructor with surface type input and properties
+      /// input. To be used for CUSTOM_SURFACE.
+      /// \param[in] _type SurfaceType specification.
+      /// \param[in] _radius Radius of the surface.
+      /// \param[in] _axisEquatorial Semi major axis of the surface.
+      /// \param[in] _axisPolar Semi minor axis of the surface.
+      /// \param[in] _axisPolar Flattening parameter of the surface.
+      public: SphericalCoordinates(
+            const SurfaceType _type,
+            const double _radius,
```

Comment:
this radius value is used as an approximate average radius for computations based on a spherical surface. I would recommend computing this radius value as the arithmetic mean of the equatorial and polar axes rather than requiring users to specify it

* see https://en.wikipedia.org/wiki/Earth_radius#Arithmetic_mean_radius

`(2 * _axisEquatorial + _axisPolar) / 3`

Reply:
Done https://github.com/gazebosim/gz-math/pull/434/commits/b8c488711b0b2b15d3283019ff937d9ebcd7886e